src/main/java/org/granitepowered/granite/GraniteStartupThread.java

```diff
@@ -87,11 +96,11 @@
     String[] args;
     BytecodeModifier modifier;
 
-    String serverVersion;
-    String apiVersion;
+    String serverVersion = "UNKNOWN";
+    String apiVersion = "UNKNOWN";
```

Comment:
These do not need to be initialised as they will be either a valid string or UNKNWN from the versionprop properties.